cluster-provision/k8s/1.28/k8s_provision.sh

```diff
@@ -301,6 +315,11 @@ kubeadm_raw_ipv6=/tmp/kubeadm_ipv6.conf
 kubeadm_manifest="/etc/kubernetes/kubeadm.conf"
 kubeadm_manifest_ipv6="/etc/kubernetes/kubeadm_ipv6.conf"
 
+# envsubst pkg is not available by default in s390x Architecture.
+if [ "$arch" == "s390x" ]; then 
+   dnf install -y gettext
```

Comment:
We could probably include this when we install this with other rpm dependencies in provision.sh

I don't think we need an if statement here - if its already installed on x86 it won't do anything anyway.